examples/_generate_assets.py

```diff
@@ -146,6 +152,14 @@
 save_and_compress_svg(fig, "ptable-heatmap-plotly-log")
 
 
+# %% Plot histograms nested within a periodic table
+elements = [Element.from_Z(z) for z in range(1, 119)]
+data_dict = {e.symbol: np.random.randint(0, 20, 20).tolist() for e in elements}
```

Comment:
I should use a more chemically meaningful data then some random data. 

Meanwhile I noticed using `Dict[str, np.ndarray]` doesn't seem to work, would see if I could fix it.

Reply:
you're calling `.tolist()` so it's no longer `np.ndarray `

Reply:
Yes `.tolist()` is added because `data_dict = {e.symbol: np.random.randint(0, 20, 20) for e in elements}` doesn't seem to work, got the following:
```bash
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
File [d:\GitHub\pymatviz\examples\_generate_assets.py:4](file:///D:/GitHub/pymatviz/examples/_generate_assets.py:4)
      [2](file:///D:/GitHub/pymatviz/examples/_generate_assets.py:2) elements = [Element.from_Z(z) for z in range(1, 119)]
      [3](file:///D:/GitHub/pymatviz/examples/_generate_assets.py:3) data_dict = {e.symbol: np.random.randint(0, 20, 20) for e in elements}
----> [4](file:///D:/GitHub/pymatviz/examples/_generate_assets.py:4) fig = ptable_hists(data_dict, colormap="coolwarm", cbar_title="Periodic Table Histograms")
      [6](file:///D:/GitHub/pymatviz/examples/_generate_assets.py:6) save_and_compress_svg(fig, "ptable_hists")

File [D:\GitHub\pymatviz\pymatviz\ptable.py:885](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:885), in ptable_hists(data, bins, colormap, hist_kwds, cbar_coords, x_range, symbol_kwargs, symbol_text, cbar_title, cbar_title_kwds, cbar_kwds, symbol_pos, log, anno_kwds, return_axes, **kwargs)
    [883](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:883)         annotation["text"] = anno_text
    [884](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:884)     ax.annotate(**(defaults | annotation))
--> [885](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:885) if hist_data:
    [886](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:886)     hist_kwargs = hist_kwds(hist_data) if callable(hist_kwds) else hist_kwds
    [887](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:887)     _n, bins_array, patches = ax.hist(
    [888](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:888)         hist_data, bins=bins, log=log, range=x_range, **(hist_kwargs or {})
    [889](file:///D:/GitHub/pymatviz/pymatviz/ptable.py:889)     )

ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

I would see if I could fix it together with #81. 

Thanks for the quick response!

Reply:
might make sense to change `if hist_data` to `if hist_data is not None`

Reply:
Thanks @janosh. Fixed as you suggested